src/coreComponents/physicsSolvers/contact/SolidMechanicsLagrangeContact.cpp

```diff
@@ -1941,27 +1944,28 @@ bool SolidMechanicsLagrangeContact::updateConfiguration( DomainPartition & domai
                 fractureState[kfe] = contact::FractureState::Stick;
               }
             }
-            checkActiveSetSub.min( compareFractureStates( originalFractureState, fractureState[kfe] ) );
+
+            changed += faceArea[kfe] * !compareFractureStates( originalFractureState, fractureState[kfe] );
+            total += faceArea[kfe];
           }
         } );
       } );
 
-      hasConfigurationConverged &= checkActiveSetSub.get();
+      changedArea += changed.get();
+      totalArea += total.get();
     } );
   } );
 
   // Need to synchronize the fracture state due to the use will be made of in AssemblyStabilization
   synchronizeFractureState( domain );
 
-  // Compute if globally the fracture state has changed
-  int hasConfigurationConvergedGlobally;
-  MpiWrapper::allReduce( &hasConfigurationConverged,
-                         &hasConfigurationConvergedGlobally,
-                         1,
-                         MPI_LAND,
-                         MPI_COMM_GEOSX );
+  // Compute global area of changed elements
+  changedArea = MpiWrapper::sum( changedArea );
+  // and total area of fracture elements
+  totalArea = MpiWrapper::sum( totalArea );
 
-  return hasConfigurationConvergedGlobally;
+  // Assume converged if changed area is below certain fraction of total area
+  return changedArea <= m_nonlinearSolverParameters.m_configurationTolerance * totalArea;
```

Comment:
does this mean that if only a small element changes state we do not update the configuration?

Reply:
My intention was to allow to converge when only a small portion of fracture elements change state
Typically happens when things are flip flopping

Reply:
Configuration should be updated, this PR should be only about convergence criteria for configuration change - currently it is strict yes or no, here we are trying to allow some tolerance